test/battle/hold_effect/rowap_berry.c

```diff
@@ -0,0 +1,44 @@
+#include "global.h"
+#include "test/battle.h"
+
+ASSUMPTIONS
+{
+    ASSUME(gItems[ITEM_KEE_BERRY].holdEffect == HOLD_EFFECT_KEE_BERRY);
```

Comment:
This file is supposed to be for rowap berry, not kee berry.

Reply:
Should be fixed.